### DISCUSSION TITLE
GrapesJS logo

### DISCUSSION THREAD

**Discussion by AitorMillan** [GrapesJS/grapesjs#5897] [Q&A] [answered]
Hi! This is my first message and i dont know if this is the right place to ask this.

I am doing my final degree work. Im ussing the editor to make the project so i would like to know if you give me permission to add your logo in my work, and if you could provide me with a SVG file with the logo in that case.

Thanks in advance, best regards.

**Comment by mosh-tudor** [comment, marked as answer]
SVG logo - https://grapesjs.com/img/logos/grapesjs-logo.svg

**Reply by AitorMillan** [reply]
Thanks! 😄
